src/Models/Activity.php

```diff
@@ -95,9 +98,4 @@ public function scopeHasBatch(Builder $query): Builder
     {
         return $query->whereNotNull('batch_uuid');
     }
-
-    public function scopeForBatch(Builder $query, string $batchUuid): Builder
```

Comment:
We consider this method as convenience method. Plus removing it now will make some tests fail.

Reply:
Ahhh - my fault. I thought it was added in for `v4` and didn't realize v4 was released already.